README.md

```diff
@@ -34,12 +34,24 @@ curl -fsSL https://pkg.gpud.dev/install.sh | sh
 
 Note that the install script doesn't support other architectures (arm64) and OSes (macos), yet.
 
-### Run locally (self-hosted option)
+### Run GPUd with Lepton Platform
 
-For linux, run the following command to start the service (self-hosted option):
+Optionally you may register your machine with Lepton AI Platform -- the managed option brings several benefits:
```

Comment:
remove line39-line43 to make the readme short and clean.

Reply:
done